### PR TITLE
zh-CN: cr2 Day 3 translations

### PR DESCRIPTION
**Pull request by henrif75**
*Stats:*
Before: 1018 translated messages, 368 fuzzy translations, 2074 untranslated messages.
After: 1143 translated messages, 383 fuzzy translations, 1934 untranslated messages.
#324 #1463

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -7028,12 +7033,12 @@ msgstr "运行时通过自动内存管理实现完全安全：Java、Python、Go
 msgid ""
 "A runtime system ensures that memory is not freed until it can no longer be "
 "referenced."
-msgstr ""
+msgstr "运行时系统可确保在无法引用内存时，不会释放该内存。"
```
Comment (emmali01) [inline po/zh-CN.po:7036]:
```suggestion
msgstr "运行时系统可确保在内存无法被引用之前，不会释放该内存。"
```

File: po/zh-CN.po [hunk 2]
```diff
@@ -7077,6 +7087,9 @@ msgid ""
 "be dereferenced, eliminating use-after-free and other classes of bugs. But, "
 "GC has a runtime cost and is difficult to tune properly."
 msgstr ""
+"Java、Go 和 Python 依赖垃圾回收器来识别无法再访问的内存并将其舍弃。这保证可对"
+"所有指针进行解引用操作，从而消除了释放后使用等各类 bug但是，垃圾回收 (GC) 会"
```
Comment (emmali01) [inline po/zh-CN.po:7091]:
```suggestion
"所有指针进行解引用操作，从而消除了释放后使用等各类 bug。但是，垃圾回收 (GC) 会"
```

File: po/zh-CN.po [hunk 3]
```diff
@@ -7123,16 +7142,16 @@ msgstr "赋值操作将在变量之间转移所有权："
 #: src/memory-management/move.md:7
 #, fuzzy
 msgid "\"Hello!\""
-msgstr "Hello World!"
+msgstr "b\"hello\""
```
Comment (emmali01) [inline po/zh-CN.po:7145]:
```suggestion
msgstr "b\"Hello!\""
```

File: po/zh-CN.po [hunk 4]
```diff
@@ -7179,24 +7199,22 @@ msgid ""
...
-":                           :  |  `- - - - - - - - - - - - - -'\n"
-":    s2                     :  |\n"
-":   +-----------+-------+   :  |\n"
-":   | ptr       |   o---+---+--'\n"
-":   | len       |     4 |   :\n"
-":   | capacity  |     4 |   :\n"
-":   +-----------+-------+   :\n"
-":                           :\n"
-"`- - - - - - - - - - - - - -'\n"
+" Stack                           Heap\n"
```
Comment (emmali01) [inline po/zh-CN.po:7202]:
```suggestion
" 栈                           堆\n"
```

File: po/zh-CN.po [hunk 5]
```diff
@@ -7349,26 +7368,30 @@ msgstr ""
...
 msgid "Clone"
-msgstr ""
+msgstr "克隆"
 
 #: src/memory-management/clone.md:3
 msgid ""
 "Sometimes you _want_ to make a copy of a value. The `Clone` trait "
 "accomplishes this."
-msgstr ""
+msgstr "有时，_如需_ 复制某个值。`Clone` trait 可以完成此操作。"
```
Comment (emmali01) [inline po/zh-CN.po:7377]:
```suggestion
msgstr "有时，_如需_ 复制某个值。`Clone` 特征 可以完成此操作。"
```

File: po/zh-CN.po [hunk 6]
```diff
@@ -7703,24 +7734,22 @@ msgid ""
...
-":                           :  |  `- - - - - - - - - - - - - -'\n"
-":    s2                     :  |\n"
-":   +-----------+-------+   :  |\n"
-":   | ptr       |   o---+---+--'\n"
-":   | len       |     4 |   :\n"
-":   | capacity  |     4 |   :\n"
-":   +-----------+-------+   :\n"
-":                           :\n"
-"`- - - - - - - - - - - - - -'\n"
+" Stack                           Heap\n"
```
Comment (emmali01) [inline po/zh-CN.po:7737]:
```suggestion
" 栈                           堆\n"
```

File: po/zh-CN.po [hunk 7]
```diff
@@ -7807,24 +7836,22 @@ msgid ""
...
```
Comment (emmali01) [inline po/zh-CN.po:7839]:
```suggestion
" 栈                           堆\n"
```

File: po/zh-CN.po [hunk 8]
```diff
@@ -8588,116 +8670,132 @@ msgid ""
...
+"`parse_varint` 的定义。该指定代码还定义了一些回调，用于处理 `Person` 和 "
+"`PhoneNumber` 字段，并将消息解析为对这些回调的一系列调用。"
 
 #: src/slices-and-lifetimes/exercise.md:38
+#, fuzzy
 msgid ""
 "What remains for you is to implement the `parse_field` function and the "
 "`ProtoMessage` trait for `Person` and `PhoneNumber`."
-msgstr ""
+msgstr "您要做的是实现 `parse_field` 函数。"
```
Comment (emmali01) [inline po/zh-CN.po:8682]:
```suggestion
msgstr "您要做的是实现 `parse_field` 函数，以及`Person` 和 `PhoneNumber`的特征`ProtoMessage`。"
```

**Review by emmali01 — APPROVED** [review]

**Review by SketchK — APPROVED** [review]

**Comment by henrif75** [comment]
Thank you @emmali01!!

**Pull request merged**